alembic/versions/9294fad3ea8f_initial_migration.py

```diff
@@ -26,11 +26,7 @@ def upgrade() -> None:
         "snippet",
         sa.Column("title", sa.String(), nullable=False),
         sa.Column("code", sa.String(), nullable=False),
-        sa.Column(
-            "language",
-            sa.Enum("JAVASCRIPT", "PYTHON", "RUST", name="language"),
-            nullable=False,
-        ),
+        sa.Column("language", sa.Text(), nullable=False),
```

Comment:
Shouldn't a db migration create a new file instead of modifying an existing one? What happened here?